06-sdks/exercises/cartojs.md

```diff
@@ -9,6 +9,18 @@
 * [Backbone.js](http://backbonejs.org/)
 * It can use either [Google Maps API](https://developers.google.com/maps/) or [Leaflet](http://leafletjs.com/)
 
+The current version of CARTO.js is the version 3 and is the version explained in this document.
+
+There are several versions available and are defined in [this section](https://github.com/CartoDB/cartodb.js/blob/2983b2fdcef914afdb1f4fdae173471143930452/scripts/publish.js#L15-L24) of the CARTO.js code. Some of them are:
```

Comment:
better refer to a release https://github.com/CartoDB/cartodb.js/blob/release-3.15.11/scripts/publish.js#L15-L24